<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FriendShow</class>
 <widget class="QDialog" name="FriendShow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
background-color: rgb(211,211,211);
border-top-left-radius : 15px;
}
QLabel
{
	font: 9pt &quot;微软雅黑&quot;;
}
</string>
  </property>
  <widget class="QTextEdit" name="signatureEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9.07563pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:9.07563pt;&quot;&gt;（签名）&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>70</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>70</width>
     <height>70</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 188, 19);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>（名字）</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="statusEdit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>在线</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>311</width>
     <height>37</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnDeleteFriend">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	font: 9pt &quot;微软雅黑&quot;;
	border-radius :7px;
	background-color: rgb(255,255,255);
}
QPushButton:hover
{
	background-color: rgb(169,189,35);
}</string>
      </property>
      <property name="text">
       <string>删除好友</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnDeleteDialog">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	font: 9pt &quot;微软雅黑&quot;;
	border-radius :7px;
	background-color: rgb(255,255,255);
}
QPushButton:hover
{
	background-color: rgb(169,189,35);
}</string>
      </property>
      <property name="text">
       <string>删除会话</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnAddBlackList">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	font: 9pt &quot;微软雅黑&quot;;
	border-radius :7px;
	background-color: rgb(255,255,255);
}
QPushButton:hover
{
	background-color: rgb(169,189,35);
}</string>
      </property>
      <property name="text">
       <string>拉黑</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="statusPoint">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>12</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(44,221,113);
border-radius :8px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>276</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>2</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="accountLabel">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>账   号</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="accountEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="ageLabel">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>年   龄</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ageEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="genderLabel">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>性   别</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="genderEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);</string>
        </property>
        <property name="currentText">
         <string>-请选择-</string>
        </property>
        <item>
         <property name="text">
          <string>-请选择-</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>男</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>女</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="cityLabel">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>城   市</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cityEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);</string>
        </property>
        <item>
         <property name="text">
          <string>-请选择-</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>北京</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>天津</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>河北</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>辽宁</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>吉林</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>黑龙江</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>山东</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>江苏</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>上海</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>浙江</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>安徽</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>福建</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>江西</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>广东</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>广西</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>海南</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>河南</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>湖南</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>湖北</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>山西</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>内蒙古</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>宁夏</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>青海</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>陕西</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>甘肃</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>新疆</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>四川</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>贵州</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>云南</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>重庆</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>西藏</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>香港</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>澳门</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>台湾</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="jobLabel">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>职   位</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="jobEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);</string>
        </property>
        <item>
         <property name="text">
          <string>-请选择-</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>教师</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>学生</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>公务员</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>律师/法务</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>计算机/通信</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>广告/传媒</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>医疗/护理</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>其他</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
